megalodon/src/mastodon/api_client.ts

```diff
@@ -606,7 +606,7 @@ namespace MastodonAPI {
       content: s.content,
       plain_content: null,
       created_at: s.created_at,
-      emojis: s.emojis.map(e => emoji(e)),
+      emojis: Array.isArray(s.emojis) ? s.emojis.map(e => emoji(e)) : [],
```

Comment:
I understood Misskey does not return this field as an array but are there any servers in Mastodon that don't return this field as an array?

Reply:
@potproject How about this? I think this PR is no problem, but I'm just wondering.

Reply:
@h3poteto 
>I understood Misskey does not return this field as an array but are there any servers in Mastodon that don't return this field as an array?

maybe no.
Also, the phenomenon of not returning the array is happening to misskey.io and not to self-host misskey.

Reply:
Additional :

There seems to be a pattern of emoji returning `{}` in the latest version of misskey v13.

The version of self-host misskey was old, so I reproduced it by updating the version.

Reply:
Wow...
I think we need to change other places to get emojis...

Reply:
@h3poteto 
Looking at misskey.io, as a difference from v12 to v13, emojis existed these patterns

* `{[key:string]:string}`
* `{}`
* not exist

There are many unknowns because this part is not documented, but it is clear that the value is unreliable.

There is another problem, but it is better than an error, so I would appreciate it if you could merge this PR to suppress the error.

Reply:
OK, I see.
Let's handle 13.0.0 emoji feature after merging this (It may not be necessary to support v12 emoji from now on).